ldpd/lde_lib.c

```diff
@@ -867,15 +870,17 @@ lde_check_withdraw(struct map *map, struct lde_nbr *ln)
 			/* LWd.9: check if previously sent a label mapping */
 			me = (struct lde_map *)fec_find(&lnbr->sent_map,
 			    &fn->fec);
+
 			/*
 			 * LWd.10: does label sent to peer "map" to withdraw
 			 * label
 			 */
-			if (me)
+			if (me && lde_nbr_is_nexthop(fn,lnbr))
```

Comment:
nit: missing space before the second argument

Reply:
fixed